channels/ir.m3u

```diff
@@ -4,7 +4,7 @@ http://51.158.190.97/hls/stream.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="4U TV" tvg-language="Persian" tvg-logo="https://i.imgur.com/Icob3E3.jpg" group-title="Entertainment",4U TV
 http://live.4ulive.ir/
 #EXTINF:-1 tvg-id="" tvg-name="Afghan TV" tvg-language="Persian" tvg-logo="https://www.parsatv.com/index_files/channels/afghantv.jpg" group-title="",AFN TV
-https://unirtmp.tulix.tv/tintv2/tintv2/playlist.m3u8
```

Comment:
Link works fine. No need to replace it.

Reply:
when I checked yesterday it was not working